openedx/core/djangoapps/content/learning_sequences/api/outlines.py

```diff
@@ -246,8 +266,6 @@ def _get_user_course_outline_and_processors(course_key: CourseKey,  # lint-amnes
         ('special_exams', SpecialExamsOutlineProcessor),
         ('visibility', VisibilityOutlineProcessor),
         ('enrollment', EnrollmentOutlineProcessor),
-        # Future:
```

Comment:
just curious -- why drop this comment? are user_partitions not needed to calculate the outline correctly?

Reply:
Yeah, they're currently used at the Unit level and below.